package.json

```diff
@@ -56,6 +56,7 @@
   },
   "dependencies": {
     "@tableland/sdk": "^3.1.5",
+    "@tableland/validator": "^0.0.0-pre.4",
```

Comment:
The binaries and the logic to identify the system needed has been extracted to a separate package so that it can be managed as a normal dependency and kept in sync with the go-tableland releases